### PR TITLE
Add error message when ordering number of item greater than quantity

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.0.x
| Description?  | When ordering a number of item greater than quantity, there is no error message
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1965
| How to test?  | Go to product detail page and add to card a number of item grater than quantity.

### CODE REVIEW

File: themes/classic/templates/catalog/_partials/product-add-to-cart.tpl [hunk 1]
```diff
@@ -53,6 +53,9 @@
                   <i class="material-icons product-unavailable">&#xE14B;</i>
                 {/if}
                 {$product.availability_message}
+              {else}
```
Comment (maximebiloe) [inline themes/classic/templates/catalog/_partials/product-add-to-cart.tpl]: Shouldn't be done in the presenter?
No more calculation into templates :wink:

**Review by maximebiloe — CHANGES_REQUESTED** [review]

File: src/Core/Product/ProductPresenter.php [hunk 2]
```diff
@@ -404,8 +404,13 @@ public function addQuantityInformation(
                 if ($product['quantity'] < $settings->lastRemainingItems) {
                     $presentedProduct = $this->applyLastItemsInStockDisplayRule($product, $settings, $presentedProduct);
                 } else {
-                    $presentedProduct['availability_message'] = $product['available_now'];
-                    $presentedProduct['availability'] = 'available';
+                    if($product['quantity_wanted'] > $product['quantity']) {
+                        $presentedProduct['availability_message'] = 'Out of stock';
```
Comment (maximebiloe) [inline src/Core/Product/ProductPresenter.php]: It's not really the same thing that "out of stock", it's more something like "we don't have enough products in stock".
Also, you forgot to make the string translatable.

**Review by maximebiloe — CHANGES_REQUESTED** [review]

File: src/Core/Product/ProductPresenter.php (hunk 2 above)
Comment (fatmaBouchekoua) [inline src/Core/Product/ProductPresenter.php]: Done, Thanks!

**Review by maximebiloe — APPROVED** [review]

**Comment by maximebiloe** [comment]
Thank you @fatmaBouchekoua

**Comment by zapto** [comment]
Doesnt work if stock management disabled. You cant add products...

**Pull request merged**